plugin.js

```diff
@@ -34,6 +34,26 @@ async function fastifyLogController (fastify, opts) {
 
   const logLevels = Object.keys(fastify.log.levels.values)
 
+  fastify.get('/log-level', {
```

Comment:
To avoid a breaking change, I would hide this route behind an option:

```
app.register(require('fastify-log-controller'), {
  exposeGet: true // default: false
})
```

Reply:
Sure!
I'll take care of it in the next few days.